### DISCUSSION TITLE
有没有带截图的debug信息？

### DISCUSSION THREAD

**Discussion by pingplug** [MaaAssistantArknights/MaaAssistantArknights#3033] [Q&A] [answered]
使用MAA刷肉鸽会出现一些奇怪问题（比如莫名其妙的死在第一个关卡），有些问题是在运行较长时间后才会发生，或者在黑屏/锁屏/模拟器窗口最小化后发生，只有文字log而不保存截图感觉还是不方便发现问题。。。
现在在MAA运行时确实会有一个放截图的文件夹，但是里面的截图内容极少，对于发现问题没啥帮助。。。
有没有什么方法能实现将每次（或者每次识别到新信息时的）截图保存的功能？
当然如果能将识别到的干员、地图坐标点、执行的操作绘制在截图上效果会更好。。。

**Comment by ABA2396** [comment, marked as answer]
感觉确实可以再多加点截图，比如任务出错的时候
